src/services/Audius/setup.ts

```diff
@@ -117,9 +117,11 @@ const configureReadOnlyLibs = async () => {
     ethProviderUrl,
     ethOwnerWallet
   )
+  const discoveryProviderConfig = {}
```

Comment:
for consistency/completeness, this probably could be config.configDiscoveryProvider() 🤷